src/internal/superjson/mutate.ts

```diff
@@ -427,6 +542,50 @@ export function addProvider(
   return true;
 }
 
+export function swapProviderVariant(
+  document: SuperJsonDocument,
+  providerName: string,
+  variant: { kind: 'local'; file: string } | { kind: 'remote' }
+): boolean {
+  if (document.providers === undefined) {
+    document.providers = {};
```

Comment:
Do we count this as "change"? 🤷

Reply:
I went with a "no" because it doesn't change the semantic view of the document, only changes `undefined` for `{}` which is semantically equal.